integ-test/src/test/java/org/opensearch/sql/sql/PaginationWindowIT.java

```diff
@@ -92,4 +96,39 @@ public void testFetchSizeLargerThanResultWindowFails() throws IOException {
         () -> executeQueryTemplate("SELECT * FROM %s", TEST_INDEX_PHRASE, window + 1));
     resetMaxResultWindow(TEST_INDEX_PHRASE);
   }
+
+  @Test
+  public void testMultiShardPagesEqualsActualData() throws IOException {
+    // A bug made it so when pulling unordered data from an index with multiple shards, data gets lost if the fetchSize
```

Comment:
> pulling unordered data from an index
it is not accurate, it is ordered data (sord by _doc)?

Reply:
Unordered meaning that there's no `ORDER BY` clause supplied by the user -- in this case we default to sorting by `_doc` but semantically the actual order doesn't matter. I wasn't there when it was written, but I suspect sorting by `_doc` is meant to line up with other SQL flavors that have a hidden `ROWID` that they use by default.